### PR TITLE
feat: add model transcriptions to gem

### PR DESCRIPTION
**Pull request by ManishNarayan**

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -1,95 +1,93 @@
...
 
 GEM
   remote: https://rubygems.org/
   specs:
-    actioncable (7.0.3.1)
-      actionpack (= 7.0.3.1)
-      activesupport (= 7.0.3.1)
+    actioncable (6.1.7)
+      actionpack (= 6.1.7)
+      activesupport (= 6.1.7)
```
Comment (rishirajkumar97) [inline Gemfile.lock:12]: bro why is rails changed from 7.0.3.1 to 6.1.7 :/

Comment (ManishNarayan) [reply Gemfile.lock:12]: It asked me to run bundle update and this happened. Not sure tho ! Will rectify

Comment (ManishNarayan) [reply Gemfile.lock]: Fixed it. It was an issue w my ruby version so it made me update bundle i think 😓

File: README.md [hunk 2]
```diff
@@ -64,6 +64,7 @@ The gem is available as open source under the terms of the [MIT License](https:/
 * Lockstep::ReportRiskRate
 * Lockstep::Status
 * Lockstep::User
+* Lockstep::TranscriptionValidationRequest
```
Comment (rishirajkumar97) [inline README.md:67]: this is an alphabetical list can we add as per the order.

Comment (ManishNarayan) [reply README.md:67]: Sure!

**Pull request merged**